appveyor.yml

```diff
@@ -109,6 +111,58 @@ build_script:
  - 7z a -tzip -r ..\output\symbols.zip *.dl_ *.ex_ *.pd_
  - cd ..
 
+before_test:
+ - py -m pip install robotframework
+ - py -m pip install robotremoteserver
+ - py -m pip install pyautogui
+ - py -m pip install nose
+ - mkdir testOutput
+ - mkdir testOutput\unit
+ - mkdir testOutput\system
+ - ps: |
+     $errorCode=0
+     $nvdaLauncherFile=".\output\nvda"
+     if(!$env:release) {
+      $nvdaLauncherFile+="_snapshot"
+     }
+     $nvdaLauncherFile+="_${env:version}.exe"

```

Comment:
Yes, the tests run on try builds, against an install of NVDA. I can't speak to your second question, it might be best for @michaelDCurran to answer that one.